src/assets/javascripts/sw.js

```diff
@@ -0,0 +1,34 @@
+const VERSION = "v2.4"
```

Comment:
I'd rather not have it pinned in the source code. `makefile` should be the only source of truth for the version (if it's absolutely needed)

Reply:
I got it. Do you have any suggestion on how should I include the version here?